new-client/src/components/Search/SearchBar.js

```diff
@@ -352,13 +352,16 @@ class SearchBar extends React.PureComponent {
     return (
       <TextField
         {...params}
-        label={undefined}
+        label={<Typography variant="srOnly">dsadsad</Typography>}
```

Comment:
Just wondering if you really want a `label` with that value on this TextField?

Reply:
Wow, that was embarrasing